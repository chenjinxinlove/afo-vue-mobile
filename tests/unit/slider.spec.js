import { mount } from '@vue/test-utils'
import Demo from '../../src/slider/demo'
import Vue from 'vue'
import Slider from '../../src/slider'

describe('Rate.vue', () => {
  // 测试是否可以被使用
  it('use', () => {
    Vue.use(Slider)
    expect(Vue.component(Slider.name)).toHaveProperty('name', 'VueComponent')
  })
  // demo测试快照
  it('demo snapshot', () => {
    expect(mount(Demo)).toMatchSnapshot()
  })
})
