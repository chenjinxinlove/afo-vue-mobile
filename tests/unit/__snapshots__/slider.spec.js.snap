// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rate.vue demo snapshot 1`] = `
<div>
  <p>基本用法</p>
  <div class="afo-slider">
    <div class="afo-slider__bar" style="width: 50%; height: 2px;"><span class="afo-slider__button"></span></div>
  </div>
  <div style="height: 50px;"></div>
  <p>指定选择范围</p>
  <div class="afo-slider">
    <div class="afo-slider__bar" style="width: 50%; height: 2px;"><span class="afo-slider__button"></span></div>
  </div>
  <div style="height: 50px;"></div>
  <p>禁操作</p>
  <div class="afo-slider afo-slider--disabled">
    <div class="afo-slider__bar" style="width: 50%; height: 2px;"><span class="afo-slider__button"></span></div>
  </div>
  <p>指定步长</p>
  <div style="height: 50px;"></div>
  <div class="afo-slider">
    <div class="afo-slider__bar" style="width: 50%; height: 4px;"><span class="afo-slider__button"></span></div>
  </div>
</div>
`;
